lib/spoom.rb

```diff
@@ -55,7 +55,7 @@ def exec(cmd, *arg, path: ".", capture_err: false)
         stdout, status = T.unsafe(Open3).capture2([cmd, *arg].join(" "), chdir: path)
         ExecResult.new(
           out: stdout,
-          err: "",
+          err: "<not captured>",
```

Comment:
What do you think about using `nil` instead? I feel like that would better inform clients that they need to do something when there's no error captured.